packages/help-center/src/hooks/use-messaging-availability.ts

```diff
@@ -0,0 +1,47 @@
+import config from '@automattic/calypso-config';
+import { useQuery } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
+import type { MessagingAvailability } from '../types';
+
+interface APIFetchOptions {
+	global: boolean;
+	path: string;
+}
+
+async function requestMessagingAvailability() {
+	const currentEnvironment = config( 'env_id' );
```

Comment:
After we trained HEs for months about proxying status, I think this will create confusion again. Can the same logic apply here? `proxied => dev, no => prod`?

Reply:
From my experience, it's more useful for HEs to be able to easily test if chat is available for production users. So I'd prefer to take this opportunity to clear this up. Especially since I imagine a lot of them will be testing this flow after we launch it and I don't want additional confusion of staging versus production 🙇